app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,36 @@
+class CategoriesController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[splash]
+
+  def index
+    @categories = current_user.ordered_groups
+  end
+
+  def new
+    group = Group.new
+    respond_to do |format|
+      format.html { render :new, locals: { group: } }
```

Comment:
- [ ] [OPTIONAL] Hello @JohnFTitor , You have a great project here, however, after dropping, creating and migrating the DB then starting the server we still have a minor error popping up due to the fact that I had to change my ruby version in the `ruby-version` file and the `Gemfile`. To stop this errordo this for line 11
From:
 `format.html { render :new, locals: { group: } }`
To:
` format.html { render :new, locals: { group:  group} }`
So that other users with different versions can also have access to it. 
### Below is the error before it got fixed:
![Screenshot from 2022-05-27 00-32-01](https://user-images.githubusercontent.com/58692590/170598197-8b0d5099-a529-4509-8522-854164cbab43.png)